client/lib/discounts/active-discounts.js

```diff
@@ -54,4 +55,22 @@ export default [
 		plansPageNoticeText:
 			'Improve your SEO, branding, credibility, and even word-of-mouth marketing with a custom domain. All plan upgrades include a free domain name of your choice.',
 	},
+	{
+		name: 'august20',
+		startsAt: new Date( 2018, 7, 6, 0, 0, 0 ),
+		endsAt: new Date( 2018, 7, 11, 0, 0, 0 ),
+		nudgeText: translate( '20% Off All Plans' ),
+		ctaText: translate( 'UPGRADE' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **76** times:
`translate( 'Upgrade' )` *ES Score: 16*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/active-discounts-august-promo)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Would have been nice if this would have been used. We already have "Upgrade" used as a CTA here: https://github.com/Automattic/wp-calypso/blob/a24b6ee28fb9b8b5747158a53a66acb309d05e8d/client/my-sites/current-site/notice.jsx#L114